### DISCUSSION TITLE
Beaver for Mobile

### DISCUSSION THREAD

**Discussion by Miserableee** [Daniele-rolli/Beaver-Notes#12] [General]
Do you have plans to publish the program for Android as well or not?

**Comment by Daniele-rolli** [comment]
Beaver Notes Pocket, the mobile version, is currently in development. Since this project is a one-person operation, the release of a proper mobile app will require some time. Additionally, it's unlikely to be available on the Play Store or the App Store, as both Apple and Google require substantial fees to publish apps on their platforms. However, it will be accessible on F-Droid and probably Aptoide, alongside the option to sideload it as soon as Apple makes it available.

**Reply by pratik-gandhi** [reply]
Hey Daniele,

First of all, thank you for your great work! Is there any way for me to contribute to the development of android/iOS versions?

**Reply by Daniele-rolli** [reply]
I have a repository called ‘Beaver Notes Pocket’ on my profile, but it’s a bit outdated. I’m currently working on a React PWA that will run on iOS and Android, using Ionic and Capacitor. As soon as I have a somewhat functional app, I’ll release my work so that everyone can contribute. The main objective right now is not to have a fully working mobile note-taking app, but to have one that is somewhat compatible with the desktop app.

**Reply by pratik-gandhi** [reply]
Great, thanks for the response. I'll keep an eye on that repo. Meanwhile, if you need any help with development/testing, do let me know. ✌️

**Reply by Daniele-rolli** [reply]
Sorry for the delay; I've been working non-stop on the mobile app. I'll upload it to GitHub with some documentation by tomorrow. If anyone who's interested happens to know React, feel free to join the project and help develop Beaver Notes Pocket.
